src/current/_includes/releases/v23.2/v23.2.13.md

```diff
@@ -0,0 +1,88 @@
+## v23.2.13
+
+Release Date: October 17, 2024
+
+{% include releases/new-release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v23-2-13-{{-site.data.products.enterprise-}}-edition-changes">{{ site.data.products.enterprise }} edition changes</h3>
+
+- The description for the [cluster setting]({% link v23.2/cluster-settings.md %}) `changefeed.sink_io_workers` now lists all sinks that support the setting. [#130372][#130372]
+- Network metrics have been added for the following [changefeed sinks]({% link v23.2/changefeed-sinks.md %}):
+      - Kafka v1
+      - Webhook
+      - PubSub
+      - SQL
+
+      [#130664][#130664]
+
+- The new [metric]({% link v23.2/metrics.md %}) `changefeed.total_ranges` allows observation of the number of changes that are watched by a changefeed aggregator. It uses the same polling interval as `changefeed.lagging_ranges`, which is controlled by the changefeed option `lagging_ranges_polling_interval`. [#130984][#130984]
+- The following groups of [metrics]({% link v23.2/metrics.md %}) and [logs]({% link v23.2/logging.md %}) have been renamed to include the buffer they are associated with. The previous metrics are still maintained for backward compatibility.
+      - `changefeed.buffer_entries_mem.*`
+      - `changefeed.buffer_pushback_nanos.*`
```

Comment:
I think you inadvertently dropped `changefeed.buffer_entries.*` from the original RN